# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../disktype/ext3-hotfix.patch
# Copyright (C) 2010 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

New mkfs.ext3 also set min_extra_isize, and thus are misqualified as ext4, ...

  - Rene Rebe <rene@exactcode.de>

--- disktype/linux.c.vanilla	2010-02-06 14:09:54.000000000 +0000
+++ disktype/linux.c	2010-02-06 14:09:18.000000000 +0000
@@ -60,9 +60,11 @@
     /* Ext4 RO_COMPAT features: HUGE_FILE, GDT_CSUM, DIR_NLINK, EXTRA_ISIZE */
     if (get_le_long(buf + 100) & 0x0078)
       fslevel = 4;
+#if 0
     /* Ext4 sets min_extra_isize even on external journals */
     if (get_le_short(buf + 348) >= 0x1c)
       fslevel = 4;
+#endif
     /* Ext4dev TEST_FILESYS flag */
     if (get_le_long(buf + 352) & 0x0004)
       is_dev = 1;
